pyproject.toml

```diff
@@ -18,8 +18,8 @@ keywords = ['exasol', 'udf', 'script-languages']
 
 [tool.poetry.dependencies]
 python = ">=3.8,<4"
-exasol_script_languages_container_tool = "^0.16.0"
-exasol-script-languages-container-ci = "^1.0.0"
+exasol_script_languages_container_tool = "^0.17.0"
```

Comment:
Why does this one use underscores and not dashes?

Reply:
Probably, copy and paste, but both seem to be ok, but I changed it to dashes.